### PR TITLE
docs/typography

### PR DESCRIPTION
**Pull request by m0ksem**
# New docs typogpragy page
image
image

Demo with borders approved by @xx13. It allows user easily find correct code example, instead of searching correct class in long example code. Title heading can also be visible now. It was hard to find it in headers mess before.

Removed useless headings, just watch and copy example!
image

Added text helpers docs.
image
Text helpers must have an interactive example I guess, but we are good to go for now.

# New demo borders
image

For typography it was hard to see where is header and header demo. So I decided to make border around demos like 100% other Frameworks have. It also will fit in interactive playground, which we want to implement later.

# Headers margins mess

We had unlogical margins set to our typography headers to satisfy docs needs. I separated docs styles and our typography module styles. Also, now headers with typography module and va content component looks the same. All values took from Figma. 

# Docs headers margins
In https://github.com/epicmaxco/vuestic-ui/issues/1479 I think what we wanted is to remove big space between title and subtitle. Made this. First subtitle was without margins before.

closes https://github.com/epicmaxco/vuestic-ui/issues/1175

### CODE REVIEW

**Comment by m0ksem** [comment]
It still doesn't look perfect. Any thoughts how we can improve these docs and made them easier to read?

**Comment by rustem-nasyrov** [comment]
@m0ksem, maybe we should add a little playground to demonstrate available utility classes?

**Comment by rustem-nasyrov** [comment]
@xx13 we need to increase text saturation for the blockquotes, because it's barely visible, any suggestions? 
image

File: packages/docs/src/components/DocsNavigation.vue [hunk 1]
```diff
@@ -51,66 +65,69 @@ const query = '?query=file=/src/App.vue'
...
-      } catch (e: any) {
-        if (e.message === 'NotAllowedError') {
-          copyButtonState.value = 'error'
-        }
-      }
-
-      setTimeout(() => { copyButtonState.value = 'default' }, 1500)
-    }
+const props = defineProps({
+  code: { type: String as PropType<string>, default: '' },
```
Comment (rustem-nasyrov) [inline packages/docs/src/components/DocsNavigation.vue:69]: As I remember we decided to not use the `PropType`, is it necessary in this property and other ones?

Comment (m0ksem) [reply packages/docs/src/components/DocsNavigation.vue:69]: Yeah. We removed them from UI lib, but not docs.

**Review by aluarius — APPROVED** [review]

File: packages/docs/src/components/DocsNavigation.vue (hunk 1 above)
Comment (asvae) [reply packages/docs/src/components/DocsNavigation.vue:69]: Probably we want to remove from both places at once. (maybe as a separate issue?)

**Review by rustem-nasyrov — APPROVED** [review]

**Comment by m0ksem** [comment]
> @xx13 we need to increase text saturation for the blockquotes, because it's barely visible, any suggestions? image

Yeah. This thing looks inappropriate 😟

**Comment by xx13** [comment]
it should have `secondary` colour which is `#767C88` in designs.

**Review by RVitaly1978 — APPROVED** [review]

File: packages/docs/src/components/DocsExample.vue [hunk 2]
```diff
@@ -125,3 +112,27 @@ export default {
   }
 }
 </style>
+
+<style lang="scss" scoped>
+  .docs-example-card {
```
Comment (asvae) [inline packages/docs/src/components/DocsExample.vue:117]: Hmmm, why lint didn't work here? Should be no tab.

**Review by asvae — APPROVED** [review]
Looks good :).

File: packages/docs/src/components/DocsExample.vue (hunk 2 above)
Comment (m0ksem) [reply packages/docs/src/components/DocsExample.vue:117]: Should be two spaces actually, like it is :)

**Pull request merged**